src/app.js

```diff
@@ -12,7 +12,7 @@ import Web3 from "./ethereum/web3";
 import * as EthereumInterface from "./ethereum/interface";
 import networkMap, {getReadOnlyRpcUrl} from "./ethereum/network-contract-mapping";
 import ipfsPublish from "./ipfs-publish";
-import Archon from "@kleros/archon";
+import Archon from "@shotaronowhere/archon";
```

Comment:
It's very weird to have a "personal" dep.

Reply:
so AFAIK, we aren't planning to continue supporting archon, it will be deprecated and v2 will use a concept of evidence display templates templates. Archon hasn't had a new release in 2 years.

we can certainly include the patch I made under the kleros repo. I don't have the permissions for that, @jaybuidl ,  what do you think about publishing an update to archon from my [fork](shotaronowhere/archon)?

This PR is intended as a quick fix to improve performance for on-going appeal rounds on gnosis chain for which loading times were > 90 seconds, and now ~30 seconds. Still not great, there are remaining performance enhancements to be made.

Reply:
Okay I made the [archon PR](https://github.com/kleros/archon/pull/68)

Reply:
created the [archon PR](https://github.com/kleros/archon/pull/68)